bazel/linux/defs.bzl

```diff
@@ -239,11 +271,47 @@ def _normalize_kernel(kernel):
 
 BUILD_LEFTOVERS = ["**/.*.cmd", "**/*.a", "**/*.o", "**/*.ko", "**/*.order", "**/*.symvers", "**/*.mod", "**/*.mod.c", "**/*.mod.o"]
 
+def _kernel_module_targets(*args, **kwargs):
+    """Common kernel module target setup."""
+
+    modules = []
+    for m in kwargs.get("modules", kwargs["name"]):
+        if not m.endswith(".ko"):
+            m = m + ".ko"
+        modules += [m]
+    kwargs["modules"] = modules
+
+    kernels = []
+    if "kernel" in kwargs:
+        kernels += [kwargs["kernel"]]
+        kwargs.pop("kernel")
```

Comment:
nit: mostly a matter of taste, but [pop](https://docs.bazel.build/versions/master/skylark/lib/dict.html#pop) returns the value that was removed, and return a default. So this could be:

```
kernels = kwargs.pop("kernels", [])
if "kernel in kwargs:
  kernels += [kwargs.pop("kernel")]
```